tensorflow/lite/kernels/activations.cc

```diff
@@ -333,11 +333,9 @@ TfLiteStatus ReluEval(TfLiteContext* context, TfLiteNode* node) {
   TfLiteTensor* output = GetOutput(context, node, 0);
   switch (input->type) {
     case kTfLiteFloat32: {
-      size_t elements = input->bytes / sizeof(float);
-      float* in = input->data.f;
-      float* in_end = in + elements;
-      float* out = output->data.f;
-      for (; in < in_end; in++, out++) *out = std::max(0.f, *in);
+      // refering to the optimized implementation
```

Comment:
No need for this comment, it's clear from the code. Same below.

Reply:
@jdduke , thanks for the review, i have updated the code as per your suggestion. Kindly check and approve.

Regards
Amit